website/docs/docs/deploy/regions-ip-addresses.md

```diff
@@ -9,12 +9,12 @@ dbt Cloud is [hosted](/docs/deploy/architecture) in multiple regions and will al
 [dbt Cloud Enterprise](https://www.getdbt.com/pricing/) plans can choose to have their account hosted in any of the below regions. Organizations **must** choose a single region per dbt Cloud account. If you need to run dbt Cloud in multiple regions, we recommend using multiple dbt Cloud accounts. 
 
 
-| Region | Location | Access URL | IP addresses | Developer plan | Team plan | Enterprise plan |
-|--------|----------|------------|--------------|-----------------|------------|------------------|
-| North America [^1] | AWS us-east-1 (N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
-| EMEA [^1] | AWS eu-central-1	(Frankfurt) | emea.dbt.com | 3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 | ❌ | ❌ | ✅ |
-| APAC  [^1] | 	AWS ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
-| Virtual Private dbt or Single tenant | Customized |  Customized | Ask [Support](/guides/legacy/getting-help#dbt-cloud-support) for your IPs | ❌ | ❌ | ✅ |
+| Region | Location | Access URL | Auth URL | Audience URN | IP addresses | Developer plan | Team plan | Enterprise plan |
+|--------|----------|------------|------------|------------|--------------|-----------------|------------|------------------|
+| North America [^1] | AWS us-east-1 (N. Virginia) | cloud.getdbt.com | us-production-mt.us.auth0.com | urn:auth0:us-production-mt | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
```

Comment:
@robotmiller after this [ticket](https://dbtlabs.atlassian.net/browse/ENTERPRISE-356), auth url will be a dbt.com domain-ed one right?

Reply:
Yes, that's correct!